source/install/desktop-app-install.rst

```diff
@@ -18,138 +18,187 @@ Desktop application install guides
+- `Add Mattermost <https://docs.mattermost.com/welcome/customize-desktop-app-experience.html>`__ to the Windows Start menu, the Taskbar, the Dock, or the System Tray.
+- (Windows/Mac) `Deep link to the Desktop App <https://docs.mattermost.com/welcome/customize-desktop-app-experience.html>`__ via the ``mattermost://`` protocol if the app is already installed.
+- (Linux) `Set up Desktop Entry <https://docs.mattermost.com/welcome/customize-desktop-app-experience.html>`__ for the application to more easily `integrate into a desktop environment <https://wiki.archlinux.org/index.php/Desktop_entries>`__.
 
-- `Guide for configuring your desktop app experience <https://docs.mattermost.com/welcome/customize-desktop-app-experience.html>`__
-- `Changelog <https://docs.mattermost.com/install/desktop-app-changelog.html>`__
+The following additional documentation resources are also available:
+
+- `Desktop App changelog <https://docs.mattermost.com/install/desktop-app-changelog.html>`__
+- `Minimum software requirements <https://docs.mattermost.com/install/software-hardware-requirements.html#desktop-apps>`__
+- `Configure your Desktop App experience <https://docs.mattermost.com/welcome/customize-desktop-app-experience.html>`__
 - `Source code <https://github.com/mattermost/desktop>`__
 - `Contributor’s guide <https://developers.mattermost.com/contribute/desktop>`__
 
-You can `download the desktop apps directly from our Downloads page <https://mattermost.com/download/>`__. You may also use the following installation guides for Windows, Mac, and Linux.
+Install the Mattermost Desktop App
+----------------------------------
+
+You can `download the Desktop App directly from our Downloads page <https://mattermost.com/download/>`__. You may also use the following installation guides for Linux, Mac, and Windows.
 
 .. include:: ../upgrade/upgrading-to-v60.rst
+  :start-after: :nosearch:
+
+.. tabs::
+
+  .. tab:: Linux
+
+    **Generic Linux package** 
+
+    1. Download the latest version of the Mattermost desktop app:
+
+      - 64-bit systems: `mattermost-desktop-5.0.4-linux-x64.tar.gz <https://releases.mattermost.com/desktop/5.0.4/mattermost-desktop-5.0.4-linux-x64.tar.gz>`__
+      - 32-bit systems: `mattermost-desktop-5.0.4-linux-ia32.tar.gz <https://releases.mattermost.com/desktop/5.0.4/mattermost-desktop-5.0.4-linux-ia32.tar.gz>`__
+
+    2. Extract the archive to a convenient location, then execute ``mattermost-desktop`` located inside the extracted directory.
+
+    3. To create a Desktop launcher, open the file **README.md** and follow the instructions in the **Desktop launcher** section.
+
+    **Ubuntu and Debian-based systems**
+
+    Unofficial, community-driven ``.deb`` packages are available.
+
+    1. Download the latest version of the Mattermost Desktop App:
+
+      - 64-bit systems `mattermost-desktop-5.0.4-linux-amd64.deb <https://releases.mattermost.com/desktop/5.0.4/mattermost-desktop-5.0.4-linux-amd64.deb>`__
```

Comment:
Need updated download URLs

Reply:
I think we have a "latest" autolink that will always grab the latest build? Not sure what it is though :P